docs/skill_development/voice-user-interface-design-guidelines/example-dialogs.md

```diff
@@ -0,0 +1,152 @@
+
+Begin by writing the example interactions of your skill from start to finish based on your Job Stories. At first don’t worry about organizing the interactions. Just try to write down as many examples as possible. If you skipped ahead and
+
+Here is an example of a First Draft Interaction of a Moon Phase Skill.
+
+|          | Example Interaction 1                                   |
+| :------- | :------------------------------------------------------ |
+| **User** | _Hey Neon, what’s the moon phase?_                      |
+| **Neon** | _Today’s moon is Waning Crescent with 55% illumination_ |
+
+| \_\_     | Example interaction 2                   |
+| :------- | :-------------------------------------- |
+| **User** | _Hey Neon, when is the next full moon?_ |
+| **Neon** | _The next full moon is on May 7th._     |
+
+| \_\_     | Example Dialog 3                                |
+| :------- | :---------------------------------------------- |
+| **User** | _Hey Neon when is the next First quarter moon?_ |
+| **Neon** | _The next First quarter moon is April 30th_     |
+
+| \_\_     | Example Dialog 4                                                   |
+| :------- | :----------------------------------------------------------------- |
+| **User** | _What phase of the moon is May 2nd?_                               |
+| **Neon** | _The moon will be Waxing Gibbous with 68% illumination on May 2nd_ |
+
+| \_\_     | Example Dialog 5                                                    |
+| :------- | :------------------------------------------------------------------ |
+| **User** | _Hey Neon, what’s the moon phase 5 days from today?_                |
+| **Neon** | _The moon will be Waning Crescent with 5% illumination on May 20th_ |
+
+| \_\_     | Example Dialog 6                                                   |
+| :------- | :----------------------------------------------------------------- |
+| **User** | _Hey Neon, can you tell me the moon phase for a certain day?_      |
+| **Neon** | _Yes, what day would you like to know the moon phase?_             |
+| **User** | _May 15th_                                                         |
+| **Neon** | _On May 15th the moon will be waning crescent and 45% illuminated_ |
+
+| \_\_     | Example Dialog 7                                                                                                                                                                                                                                                                                                                                      |
+| :------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
This block of dashes is much larger than in the other examples, I'm wondering if it's correct like this or ought to be reduced.